src/HttSystematics_MSSMvsSMRun2.cc

```diff
@@ -2817,7 +2834,7 @@ void AddMSSMvsSMRun2Systematics(CombineHarvester &cb, bool jetfakes, bool embedd
   cb.cp()
       .channel({"et", "mt", "tt", "em"})
       .process({"TTT", "TTL", "TTJ", "TT"})
-      .AddSyst(cb, "CMS_htt_ttbarShape", "shapeU", SystMap<>::init(1.00));
+      .AddSyst(cb, "CMS_htt_ttbarShape", "shape", SystMap<>::init(1.00));
```

Comment:
Is this change intended? I think probably not as this would change the treatment of the uncertainty for all fits and not only for the plots were it is necessary for correct uncertainty bands.